AuxTel/Troubleshooting/General-Troubleshooting/AuxTel-Rotator-Out-Of-Range.rst

```diff
@@ -0,0 +1,105 @@
+    File "/net/obs-env/auto_base_packages/ts_observatory_control/python/lsst/ts/observatory/control/base_tcs.py", line 615, in slew_icrs await self.slew( 
+    File "/net/obs-env/auto_base_packages/ts_observatory_control/python/lsst/ts/observatory/control/base_tcs.py", line 814, in slew raise ack_err 
+    File "/net/obs-env/auto_base_packages/ts_observatory_control/python/lsst/ts/observatory/control/base_tcs.py", line 803, in slew await self._slew_to( 
+    File "/net/obs-env/auto_base_packages/ts_observatory_control/python/lsst/ts/observatory/control/auxtel/atcs.py", line 1413, in _slew_to await slew_cmd.start(timeout=slew_timeout) 
+    File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-6.0.0/lib/python3.10/site-packages/lsst/ts/salobj/topics/remote_command.py", line 487, in start return await cmd_info.next_ackcmd(timeout=timeout) 
+    File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-6.0.0/lib/python3.10/site-packages/lsst/ts/salobj/topics/remote_command.py", line 191, in next_ackcmd raise base.AckError(msg="Command failed", ackcmd=ackcmd) 
+    lsst.ts.salobj.base.AckError: msg='Command failed', ackcmd=(ackcmd private_seqNum=1630455441, ack=<SalRetCode.CMD_FAILED: -302>, error=6611,  
+    result='Rejected : Rotator out of range. Target in rotator limit (-170 to 170 degrees) but out of slew limit margin (1 degs)') 
+
+
+
+.. _AuxTel-Rotator-Out-Of-Range-Procedure-Steps:
+
+Procedure Steps
+===============
+
+The interim solution requires recovering the ATPtg back to ``ENABLE`` in the case it went to ``FAULT``, and skiping to the next target in the ATQueue by pressing the :guilabel:`PLAY` buttom. 
+If this script also fails with the same error, wait three mintues and try again.
+
+
+1. Recover ATPtg from ``FAULT`` state: Transition the ATPtg CSC from LOVE ASummaryState through the states, ``FAULT`` → ``STANDBY`` → ``START`` → ``ENABLE``
+2. Add a `correct_pointing.py` script to the queue to recover the pointing offsets from the beginning of the night. No configuration is needed, you can use the default configuration.  
+3. Press :guilabel:`PLAY` button in the ATQueue: This will skip to the next target waiting in the queue.
+4. If this "new" script fails as well, wait three minutes and repeat 1-2 again.
+5. Log in relevant information about this failure in the ticket `OBS-52 <https://rubinobs.atlassian.net/browse/OBS-52>`__ - Unable to retrieve issue.
+
+
+Alternative Procedure
+=====================
+
+In case previous steps did not work, rotator continues failing and ATPtg never went to ``FAULT`` state, there is an alternative solution as follows:
+
+1. Find a target with the standard script `auxtel/track_target.py` and the following configuration:
+
+- az: 90.0
+- el: 60.0
+- mag_limit: 8.0
+
+1. Then run the external `auxtel/correct_pointing.py` on the same area of the sky using the configuration. Modify *az* and *el* accordingly:
+
+- az: 90.0
+- el: 60.0
+
```

Comment:
What to do after this? Keep playing queue. Wouldn't it go to the same area again of the sky and go into out of range again?

Reply:
Yes, just continue the queue, I added this now, thx.
About your other questions, Mmm...I think I'm not entirely sure what you mean. What you say makes sense to me, but in the instances we've encountered this issue, we've followed the same solution and it has worked. You could be right, but so far, the procedure has been effective.
Gonna go deeper to see if I can find something else about that.